courses/machine_learning/intro_to_tensorflow/2_dataset_api.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>typo, missing 'n': "patter" to "pattern"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/548/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Done!</p>